src/interface/src/app/services/plan.service.ts

```diff
@@ -237,6 +240,19 @@ export class PlanService {
     this.planState$.next(updatedState);
   }
 
+  updateStateWithScenario(scenarioId: string | null) {
```

Comment:
I'm trying to remember why we save plan state, since plans are navigable by URL now. What is plan state used for and can we get rid of it?

Reply:
Originally plan state was there so it could potentially be passed around the components. We currently just pass via parent-child in the plan component. Scenario state is needed though, because scenario is a child route (https://stackoverflow.com/questions/37662456/angular-2-output-from-router-outlet)